<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Messenger-v0.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(170, 255, 127);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btn_dark">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(99, 99, 99);</string>
       </property>
       <property name="text">
        <string>Dark Theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_del_all">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(243, 255, 116);</string>
       </property>
       <property name="text">
        <string>Delete Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ref">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(152, 152, 226);</string>
       </property>
       <property name="text">
        <string>Referesh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrl">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>480</width>
        <height>419</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QGridLayout" name="gl_messages"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="tb_name">
       <property name="sizeIncrement">
        <size>
         <width>1</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>Type your name here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tb_text">
       <property name="placeholderText">
        <string>Type your message here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_send">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 170, 255);</string>
       </property>
       <property name="text">
        <string>send</string>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
